Dockerfile

```diff
@@ -1,13 +1,23 @@
 FROM python:3.11.6-slim
 
 RUN mkdir -p /opt/dagster/dagster_home /opt/dagster/app
-RUN apt-get update && apt-get install -y libeccodes-tools
-RUN apt-get update && apt-get install -y zip
+# Some poetry packages require the following installations
+RUN apt-get update && apt-get install -y \
+    libeccodes-tools \
+    zip \
+    gcc \
+    g++ \
+    binutils \
+    libproj-dev \
+    gdal-bin \
+    libgdal-dev
 
 ENV POETRY_VIRTUALENVS_CREATE=false
 WORKDIR /opt/dagster/app
 RUN pip install poetry
 COPY pyproject.toml poetry.lock /opt/dagster/app/
+
+RUN poetry config --local installer.no-binary rasterio,fiona
```

Comment:
What does this do?

Reply:
Will add a comment. It ensures that it does not install the binaries for rasterio and fiona. They are not needed, and makes the installation crash on my m1 mac.